bots/mainbot/pilgrim.js

```diff
@@ -251,24 +248,24 @@ pilgrim.move = (self, target) => {
 	if (pilgrim.path.length === 0) {
 		if (utilities.inMovementRange(self, target)) {
 			// Target in movement range, so we can move one square towards target
-			let dx = target.x - self.x;
-			let dy = target.y - self.y;
-			let step = {x: self.x, y: self.y};
+			let dx = target.x - self.me.x;
+			let dy = target.y - self.me.y;
+			let step = {x: self.me.x, y: self.me.y};
```

Comment:
Wait, was this the bug you were referring to?

Reply:
Yeah. This entire block of code was completely messed up.